sdk/tables/azure-data-tables/README.md

```diff
@@ -231,12 +257,12 @@ the client level to enable it for all requests.
 
 ## Troubleshooting
 ### General
```

Comment:
The Troubleshooting section should have soemthing about how to enable logging. See  Jose's README for an example: https://github.com/Azure/azure-sdk-for-js/blob/56cf00b94e51444a89c96b65b6ec70a29ca6739d/sdk/tables/data-tables/README.md